site/en/install/lang_java.md

```diff
@@ -1,207 +1,203 @@
-# Install TensorFlow for Java
+# Install TensorFlow Java
 
-TensorFlow provides a
-[Java API](https://www.tensorflow.org/api_docs/java/reference/org/tensorflow/package-summary)—
-particularly useful for loading models created with Python and running them
-within a Java application.
+[TensorFlow Java](https://github.com/tensorflow/java) can run on any JVM for building, training and 
+deploying machine learning models. It supports both CPU and GPU execution, in graph or eager mode, and
+presents a rich API for using TensorFlow in a JVM environment. Java and other JVM languages, like Scala
+and Kotlin, are frequently used in small-to-large enterprises all over the world, which makes TensorFlow 
```

Comment:
"small-to-large enterprises" is a bit odd, an English idiom would be "large and small enterprises" or "enterprises of all sizes".

Maybe this line should say "TensorFlow Java" rather than just "TensorFlow".